code/game/machinery/bots/medbot.dm

```diff
@@ -46,10 +46,10 @@
 	name = "\improper Mysterious Medibot"
 	desc = "International Medibot of mystery."
 	skin = "bezerk"
-	treatment_oxy = "omnizine"
-	treatment_brute = "omnizine"
-	treatment_fire = "omnizine"
-	treatment_tox = "omnizine"
+	treatment_oxy = "tricordrazine"
+	treatment_brute = "tricordrazine"
+	treatment_fire = "tricordrazine"
+	treatment_tox = "tricordrazine"

```

Comment:
agreed actually, i love trekchems as much as the next guy but you'll rarely see more than one mysterious medibot so they should never be ODing you